site/plugins/site/helpers/html.php

```diff
@@ -13,7 +13,17 @@ function ariaCurrent(
 
 function icon(string $name): string|false
 {
-	return svg('assets/icons/' . $name . '.svg');
+	if ($svg = svg('assets/icons/' . $name . '.svg')) {
+		return $svg;
+	}
+
+	if ($panel = svg('kirby/panel/dist/img/icons.svg')) {
```

Comment:
Caching could be as simple as:

```suggestion
	static $panel;
	$panel ??= svg('kirby/panel/dist/img/icons.svg');
	if ($panel) {
```

Reply:
Thanks, added!